Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -827,12 +829,14 @@ void TextEditor::keydown_event(KeyEvent& event)
     if (event.modifiers() == Mod_Shift && event.key() == KeyCode::Key_Delete) {
         if (m_autocomplete_box)
             m_autocomplete_box->close();
+        event.ignore();
         return;
     }
 
     if (event.key() == KeyCode::Key_Delete) {
         if (m_autocomplete_box)
             m_autocomplete_box->close();
+        event.ignore();
         return;
     }
```

Comment:
These are so weird. Why does text editor not handle `Key_Delete` itself in the first place? 🤔